.github/workflows/ci.yml

```diff
@@ -53,3 +53,60 @@ jobs:
         run: |
           cargo test --all
           cargo test --all --all-features
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        args:
+          - ""
+          - "-F csv"
+          - "-F flatgeobuf"
+          - "-F flatgeobuf_async"
+          - "-F geozero"
+          - "-F ipc_compression"
+          - "-F parquet"
+          - "-F parquet_async"
+          - "-F parquet_compression"
+          - "-F polylabel"
+          - "-F postgis"
+          - "-F rayon"
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          submodules: 'recursive'
+      - uses: dtolnay/rust-toolchain@stable
+      - uses: Swatinem/rust-cache@v2
+      - name: Test
+        run: cargo build ${{ matrix.args }}
```

Comment:
We might be able to just use `cargo check` here.

A note that in the long term we could use https://github.com/frewsxcv/cargo-all-features, but with many feature combinations that can make CI _very long_.

Reply:
Switched to `check` for now, but if this isn't what you want we can switch it back

Reply:
Nope, makes sense to me, thanks.